sunkit_pyvista/plotter.py

```diff
@@ -58,6 +64,16 @@ def show(self, *args, **kwargs):
         """
         self.plotter.show(*args, **kwargs)
 
+    def _add_mesh_to_dict(self, block_name, mesh):
+        """
+        Adds all of the meshes to a `dict`
+        that stores a reference to the meshes.
+        """
+        if block_name in self.all_meshes:
+            self.all_meshes[block_name].append(mesh)
```

Comment:
I was wondering if using a default dict with a list would make this simpler. But maybe its overkill.

Reply:
I thought of that too, but default dict wasn't really used anywhere in sunpy as well so I thought it wasn't required, maybe an upgrade later on?

Reply:
Sure.